<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alterarhsiForm</class>
 <widget class="QWidget" name="alterarhsiForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>531</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Esta função realiza transformações no espaço de cor HSI. 
Use os respectivos deltas para alterar os valores de Hue,Saturation e Intensity.
Após as transformações a imagem é convertida de volta para o espaço RGB.
O valor de H é medido em graus e pode variar de 0° a 360°. Os valores de S e I variam de -255 a +255.</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>102</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSpinBox" name="rec">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDial" name="inputalterarhsiFormDeltaH">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>78</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>78</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QDial{
	background : darkgray;
	selection-background-color : darkred;
}
</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhPreferNumbers|Qt::ImhSensitiveData</set>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="invertedAppearance">
       <bool>true</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="wrapping">
       <bool>true</bool>
      </property>
      <property name="notchTarget">
       <double>45.000000000000000</double>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Delta H</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>101</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSpinBox" name="inputalterarhsiFormDeltaS">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>-255</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="verticalSlider">
      <property name="cursor">
       <cursorShape>ClosedHandCursor</cursorShape>
      </property>
      <property name="minimum">
       <number>-255</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>25</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Delta S</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>101</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSpinBox" name="inputalterarhsiFormDeltaI">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>-255</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="verticalSlider_2">
      <property name="cursor">
       <cursorShape>ClosedHandCursor</cursorShape>
      </property>
      <property name="minimum">
       <number>-255</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
      <property name="tickInterval">
       <number>25</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Delta I</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>inputalterarhsiFormDeltaH</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rec</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rec</sender>
   <signal>valueChanged(int)</signal>
   <receiver>inputalterarhsiFormDeltaH</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputalterarhsiFormDeltaI</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>inputalterarhsiFormDeltaI</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>inputalterarhsiFormDeltaS</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inputalterarhsiFormDeltaS</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
